nx/include/switch/applets/psel.h

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file psel.h
+ * @brief Wrapper for using playerSelect (user selection applet).
+ * @author XorTroll
+ * @copyright libnx Authors
+ */
+#pragma once
+#include "../types.h"
+#include "../services/applet.h"
+#include "../services/acc.h"
+
+/// playerSelect UI modes.
+typedef enum {
+    PselUiMode_SelectUser                           = 0,  ///< Simple user selection (new users cannot be created).
+    PselUiMode_UserCreation                         = 1,  ///< Only user creation (the user is later returned).
+    PselUiMode_EnsureNsaAvailable                   = 2,  ///< EnsureNsaAvailable
+    PselUiMode_IconEditor                           = 3,  ///< IconEditor
+    PselUiMode_NicknameEditor                       = 4,  ///< NicknameEditor
+    PselUiMode_ForStarter                           = 5,  ///< Mode "starter" uses to register the console's first user on the initial setup
+    PselUiMode_NetworkServiceAccountRegistration    = 8,  ///< NetworkServiceAccountRegistration
+    PselUiMode_NintendoAccountNnidLinker            = 9,  ///< NintendoAccountNnidLinker
+    PselUiMode_LicenseRequirementsForNetworkService = 10, ///< LicenseRequirementsForNetworkService
+    PselUiMode_NaLoginTest                          = 12, ///< NaLoginTest
+} PselUiMode;
+
+/// UI settings for playerSelect.
+typedef struct {
+    u32 mode;                                                   ///< UI mode, see \ref PselUiMode.
+    u32 dialogType;                                             ///< Dialog type
+    union { u128 invalidUserList[ACC_USER_LIST_SIZE]; } PACKED; ///< List of user IDs which will be disabled.
+    u8 unk_x0[0x8];                                             ///< Unknown.
+    u8 networkServiceRequired;                                  ///< Whether the user needs to be linked to a Nintendo account.
+    u8 unk_x1[0x2];                                             ///< Unknown.
+    u8 allowUserCreation;                                       ///< (With ::PselUiMode_SelectUser) enables the option to create a new user.
+    u8 skipEnabled;                                             ///< Enables the option to skip user selection (a new button is shown)
+    u8 unk_x2[0xB];                                             ///< Unknown.
+} PselUiSettings;
+
+/// Result data sent after execution.
+typedef struct {
+    u32 result;                      ///< Result code.
+    union { u128 userId; } PACKED;   ///< Selected user ID.
```

Comment:
Use AccountUid here too.

Reply:
Ah, didn't know it has a custom type now